nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -546,6 +546,28 @@ def quantize_output(self, output, wrapper, **kwargs):
         out = self._dequantize(module, out)
         return out
 
+    def load_calibration_config(self, calibration_config):
+        modules_to_compress = self.get_modules_to_compress()
+        for layer, config in modules_to_compress:
+            name, module = layer.name, layer.module
+            if name not in calibration_config:
```

Comment:
I think it is better to separate `weight` with `input`. For example, Given layer like `conv1-bn-relu-conv2`, The input of `conv2` is equal to `relu`'s output. And we should only set once for this tensor in the configure list. If it is set as the output of `relu`, we do not need to configure the quantization of the input of the `conv2`

Reply:
I agree with the conclusion that we should separate weight and input. The reason why I only judge `weight_bits` and `output_bits` while ignore `input_bits` is that input quantization will always appear with weight or output quantization, it will not appear alone. Currently I add `input_bits` to avoid misunderstanding.